client/my-sites/stats/stats-purchase/stats-purchase-commercial.tsx

```diff
@@ -0,0 +1,64 @@
+/* eslint-disable jsx-a11y/anchor-is-valid */
+import { Button } from '@wordpress/components';
+import { useTranslate } from 'i18n-calypso';
+import { COMPONENT_CLASS_NAME } from './stats-purchase-wizard';
+interface CommercialPurchaseProps {
+	planValue: number;
+}
+
+const CommercialPurchase = ( { planValue }: CommercialPurchaseProps ) => {
+	const translate = useTranslate();
+
+	return (
+		<div>
+			<div className={ `${ COMPONENT_CLASS_NAME }__notice` }>
+				{ translate(
+					'Upgrade now to take advantage of the introductory flat rate. Starting in 2024, we will introduce metered billing. '
+				) }
+				<Button variant="link" href="#">
+					{ translate( 'Learn more' ) }
+				</Button>
+			</div>
+
+			<div className={ `${ COMPONENT_CLASS_NAME }__pricing` }>
+				<div className={ `${ COMPONENT_CLASS_NAME }__pricing-value` }>
+					<div className={ `${ COMPONENT_CLASS_NAME }__pricing-currency` }>$</div>
+					<div className={ `${ COMPONENT_CLASS_NAME }__pricing-amount` }>{ `${ planValue }` }</div>
+				</div>
+				<div className={ `${ COMPONENT_CLASS_NAME }__pricing-cadency` }>
+					/{ translate( 'month' ) }
+				</div>
+			</div>
+
+			<div className={ `${ COMPONENT_CLASS_NAME }__benefits` }>
```

Comment:
We could potentially reuse the features on the checkout page:

https://github.com/Automattic/wp-calypso/blob/4bed13fb704fcc9652148c540c1cd8750f2616af/client/my-sites/checkout/composite-checkout/lib/get-jetpack-product-features.ts#L207

Reply:
Good idea. I added one missing feature description. 
Should we add the opposite 'no-' features to free stats? I don't think we can the reversed "x" icon.

Reply:
I'm happy as is. Let's consider this later. 👍